### ISSUE TITLE
Reinstalling Edge after removing with AveYo EdgeRemover, results in it still detecting edge as uninstalled, greying out options

### ISSUE THREAD

**Issue by Axer128** [rcmaehl/MSEdgeRedirect#170]
### Preflight Checklist

- [X] Microsoft Edge is still installed (see issue 26)
- [X] Running microsoft-edge:https://google.com redirects successfully
- [X] Microsoft Edge is selected in any "How do you want to open this?" box (if applicable)

### Install Type

New Deployment (Chocolatey, Winget, Etc)

### Install Mode

Active Mode

### Steps to reproduce

Fairly fresh Windows 11 install (21H2, Build 22000)
Edge was working fine but figured I had no need for it, so removed with AveYo EdgeRemover.
Installed MSEedgeRedirect, it correctly detected aveyo remover and seemed to work, tho did not test extensively.
Encountered some unrelated problems that i think needed edge installed to fix, so reinstalled edge via microsoft store. It installs no problem, but upon running I get the error popup "Edge was updated, need to run *ifeo.exe to update. I click okay and it says that file is not found. Searched entire system, it does not exist. Googled to download it, does not exist anywhere"

So after this, the redirect was broken. Most actions would either just flash the msedgeredict in the systray and do nothing. Log would claim the url is invalid, no matter what the url.

Edge also would not start via msedge.exe as it would always complain this must be updated first. However going into the version numbered directory than running msedge.exe, Edge worked fine.

-Tried reinstalling edge via choco instead. No change, it also does not install any ifeo.exe. 
-Tried reinstall msedgeredirect. It always detects edge as uninstalled via aveYo, with all the other installed options greyed out.
-Tried both active and service modes, no difference.

Gave up on it, few weeks later after edge forcibly updated itself, I noticed because my direct shortcut broke itself since it's meant to link to the stub msedge.exe, not the version numbered one which gets deleted after updates.. I went back into the program files (x86) and found it did have the ifeo file in there this time. Accepted the update after prompted and edge now works fine. So seems that required files is only installed after an update , Or not sure if it's something your program only generates, but fails to do so on a install after aveyo uninstall, and only on updates? Dunno.

Either way edge was working fully normally. So I re-installed MSEedgeRedirect yet again, choosing active mode, and while I can't select the correct "edge stable" option, it's greyed out still.. It otherwise seems to mostly function as intended.

start - run - microsoft-edge:https://google.com/ redirects successfully

Though some other ways windows 11 tries to run edge still fail, may or may be related to this issue:
Start - settings - privacy - general - managing privacy settings and changing privacy settings help links both open up edge, but then MSEedgeRedirect flashes in the system tray, closing edge. So they do not open in chrome. (Related issue: 
https://github.com/rcmaehl/MSEdgeRedirect/issues/143 )
Clicking the unremovable microsoft 365 view benefits ad in settings does open in chrome, tho i believe thats default behavior without this.
I haven't tested all links, but i think most links in settings app do not redirect. Also tested:
System - display - help with display, all 4 links broken
System - sound - help with sound

### ✔️ Expected Behavior

MSEedgeRedirect should correctly detect Edge is re-installed, not caring it was previously removed by aveyo's edge remover.

Question:
How does it detect I ran the script? Can I clean up my system now manually to have it not detect it? Registry key I need to delete or  file?

### ❌ Actual Behavior

MSEedgeRedirect incorrectly detects edge is not uninstalled and can not override it to tell it edge stable is installed. Doesn't care if it was installed via microsoft store, or chocolatey .

This results in 100% broken behavior, until an app update is triggered, in which case the user can than manually intervene to solve.

### Microsoft Windows version

Up to date Stable Windows 11 install (21H2, Build 22000)

**Comment by Axer128** [comment]
Anyone know how the detection works?
I found some files related in C:\ProgramData, deleted them but no change, it still knows i used aveyo's edge remover (but edge is installed and working)

**Comment by rcmaehl** [comment]
> Anyone know how the detection works? I found some files related in C:\ProgramData, deleted them but no change, it still knows i used aveyo's edge remover (but edge is installed and working)

Please use the latest test build if you aren't already. Detection is done via some registry keys, mainly

`HKLM\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Image File Execution Options\ie_to_edge_stub.exe`

**Comment by Axer128** [comment]
Deleting this key seems to have solved it:
HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Image File Execution Options\msedge.exe\MSER5

(It referenced ie_to_edge_stub.exe , which is no longer on my machine,so removed it)

I'll still test the self-compiled au3 as requested though.

**Comment by Axer128** [comment]
Tried the self compiled. No changes in functionality. Running MSER settings said .exe not found. It was no longer in program data, I fixed the shortcut to C:\Program Files\MSEdgeRedirect where the new .exe was. It works but would nag everytime about an update being available, as it believed it's version number was 0.0.0.0 . It also had no icon.

So I tried the build linked in the readme.md (testing build passing), and it works, tagged 0.7.0.2
It gives the message 
"You're running a newer build build than public Available!"
, with only an okay choice, rather than a yes/no as the .au3 version.

Which is only semi true.. I mean anyone in the public can access this build. But yes it's newer the an releases tab 0.7.0.1.

Don't think either of these annoying popup prompts are necessary. A little notice inside the GUI about the current version found on github vs whats running would suffice.

BTW , not releted to this ticket, but the UI looks a little bad on my setup:
<img width="480" alt="image" src="https://user-images.githubusercontent.com/63986569/192289477-35d27648-137e-4e96-9adb-c4df0e2c862f.png">

Light grey font on white.. Doesn't conform to windows common control colors. I do use a custom dark mode theme, a fairly popular one for windows 11:
https://cleodesktop.com/full-dark-theme-for-windows-11/

But 99% of apps look fine. I think your custom styling the controls, when it's better to just use the defaults, so the users theme applies normally.
